packages/f-content-cards/src/components/cardTemplates/HomeContentCard.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div
+        :data-test-id="testId"
+        :style="{ background: backgroundColor }"
+        :class="['c-homeContentCard', $style['c-homeContentCard']]"
+    >
+        <div
+            :data-test-id="containerTestId"
+            :class="['l-container', $style['c-homeContentCard-container']]"
```

Comment:
Sorry, I meant to add more global stylable classes in each of these:
```suggestion
            :class="['c-homeContentCard-container', 'l-container', $style['c-homeContentCard-container']]"
```